components/uuf-renderablecreator-hbs/src/main/java/org/wso2/carbon/uuf/renderablecreator/hbs/impl/js/JsFunctionsImpl.java

```diff
@@ -159,10 +160,13 @@ public DestroySessionFunction getDestroySessionFunction() {
 
     public SendToClientFunction getSendToClientFunction() {
         if (sendToClientFunction == null) {
-            sendToClientFunction = (name, value) -> {
-                String scriptTag = "<script type=\"text/javascript\">var " + name + "=" + GSON.toJson(value) +
+            sendToClientFunction = (name, values) -> {
+                String scriptTag = "<script type=\"text/javascript\">var " + Handlebars.Utils.escapeExpression(name) +
```

Comment:
Haa i See this this is related #101, in my opinion it should always go to head, no options needed.